### ISSUE TITLE
impl Mul<jubjub::ExtendedPoint> for jubjub::Fr and upstream.

### ISSUE THREAD

**Issue by dconnolly** [ZcashFoundation/zebra#345]
So we can reverse this to match the math in the spec / general scalar mult notation convention.

```rust
impl From<ProofAuthorizingKey> for NullifierDerivingKey {

    fn from(nsk: ProofAuthorizingKey) -> Self {
        let generator_point = zcash_h();

        Self(jubjub::AffinePoint::from(generator_point * nsk.0))
    }
}
```

![image](https://user-images.githubusercontent.com/552961/78465382-f0912b00-76c2-11ea-9443-77f613ca8aba.png)

**Comment by teor2345** [comment]
@dconnolly does this belong in the "Transaction Validation" milestone?

**Comment by mpguerra** [comment]
I keep coming across this issue and I'm not sure if this is something that needs to be done before NU5 testnet activation... @dconnolly can you help me understand? :)

**Comment by dconnolly** [comment]
> I keep coming across this issue and I'm not sure if this is something that needs to be done before NU5 testnet activation... @dconnolly can you help me understand? :)

This is not required for NU5, this is a nice-to-have

**Comment by dconnolly** [comment]
This is a purely nice-to-have and I don't care much about it anymore so closing
